docs/posts/lms-for-data/index.qmd

```diff
@@ -0,0 +1,691 @@
+---
+title: "Using language models for data"
+author: "Cody Peterson"
+date: "2024-02-05"
+image: thumbnail.png
+categories:
+    - blog
+    - llms
+    - duckdb
+---
+
+## Overview
+
+This post will give an overview of how (large) language models (LMs) fit into
+data engineering, analyst, and science workflows.
+
+## Use cases
+
+There are three main use cases for language models for data practitioners:
+
+1. Synthetic data generation
+2. Natural language processing
+3. Writing analytic code
+
+We'll describe each and then demonstrate them with code.
+
+## Setup
+
+We'll use [Marvin](https://askmarvin.ai), the AI engineering toolkit, alongside
+[Ibis](https://ibis-project.org), the data engineering toolkit, to demonstrate
+the capabilities of language models for data using the default DuckDB backend.
+
+:::{.callout-tip}
+We'll use a cloud service provider (OpenAI) to demonstrate these capabilities.
+In a follow-up post, we'll explore using local "open source" language models to
```

Comment:
```suggestion
In a follow-up post, we'll explore using local open source language models to
```

The quotes feel a bit out of place here to me.

Reply:
fair enough. point is the models aren't really open source, but I'm not explaining that here. will go into that in the follow up post